internal/pkg/service/common/errors/statuscode.go

```diff
@@ -1,28 +1,44 @@
 package errors
 
 import (
+	"context"
 	"net/http"
 
 	goa "goa.design/goa/v3/pkg"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
+const (
+	StatusClientClosedRequest = 499
+)
+
 type withStatusCode struct {
 	error
 	httpCode int
 }
 
 func HTTPCodeFrom(err error) int {
```

Comment:
Funkcia `HTTPCodeFrom` urcuje HTTP code na zaklade chyby.
Podla toho sa potom rozhoduje ci je error - server error (5xx), alebo client error (4xx).
Ci sa to ma zalogovat ako chyba, alebo ako warning a pod.